.travis.yml

```diff
@@ -3,36 +3,40 @@ language: python
 python: 3.7
 cache: pip
 install:
-  - wget https://github.com/Sage-Bionetworks/infra-utils/archive/master.zip -O /tmp/infra-utils.zip
-  - unzip -j -n /tmp/infra-utils.zip -x "infra-utils-master/.gitignore" "infra-utils-master/LICENSE" "infra-utils-master/*.md" "infra-utils-master/aws/*"
-  - ./setup_aws_cli.sh || travis_terminate 1
   - ./install_packer.sh || travis_terminate 1
   - pip install pre-commit travis-wait-improved ansible
 before_script:
   - path=$(pwd)
   - export REPO_NAME="${path##*/}"
+# Setup AWS CLI profile
+  - mkdir -p ~/.aws
+  - echo -e "[default]\nregion=us-east-1\nsource_profile=default\nrole_arn=$AwsCfServiceRoleArn" > ~/.aws/config
+  - echo -e "[default]\nregion=us-east-1\naws_access_key_id=$AwsTravisAccessKey\naws_secret_access_key=$AwsTravisSecretAccessKey" > ~/.aws/credentials
+# Setup packer params
   - export IMAGE_NAME="${REPO_NAME}-${TRAVIS_BRANCH}"
   - export AWS_PROFILE="default"
-  - export AWS_REGION="us-east-1"
+  - export AWS_SOURCE_IMAGE_PATH="/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2"
 stages:
   - name: validate
   - name: deploy-branch
     if: type = push AND branch = master
   - name: deploy-tag
-    if: tag =~ ^v
+    if: tag =~ /^v([0-9]+)\.([0-9]+)\.([0-9]+)(?:([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+[0-9A-Za-z-]+)?$/
 jobs:
   include:
     - stage: validate
       script:
         - pre-commit run --all-files
         - pushd src
         - echo "ImageName = ${IMAGE_NAME}"
-        - packer validate -var ImageName=${IMAGE_NAME} template.json
+        - packer validate -var ImageName=${IMAGE_NAME} -var SourceImage="some_aws_image_path" template.json
```

Comment:
Why is this addition necessary?

Reply:
i updated the packer file to take a `SourceImage` as input.  for validating we just need to pass in a dummy value.